### ISSUE TITLE
Can’t downloads the artifacts

### ISSUE THREAD

**Issue by nononymousse** [UMSKT/UMSKT#89]
hi ^^

i wanted to download the macOS version but the last successful compilation was 3 monts ago and thus the artifact expired so we are not able to download it :/ (other targets have that issue too)

can you do something about it ? thank you

**Comment by thepwrtank18** [comment]
Started a new workflow run, check again in around 15 minutes at most.

**Comment by thepwrtank18** [comment]
https://github.com/UMSKT/UMSKT/actions/runs/8411069210

**Comment by nononymousse** [comment]
thank you, but really, you (the project) should really use the release tab, so it won’t happen again :/ (even if it’s not perfect it’s totally okay)

**Comment by Neo-Desktop** [comment]
yeah that will be the case going forward, we should have official releases soon

in the meantime contributors should be able to re-run the CI script on the master branch to refresh the artifact

**Comment by thepwrtank18** [comment]
best way to do it besides official releases would be to have a bot that runs the CI script every 2 or so months

**Comment by nononymousse** [comment]
thanks a lot ^^
personally the releases would be better also because you need a GitHub account to download the artifacts and not everyone has one

**Comment by nononymousse** [comment]
hi ^^
so i tried to run it but i think it’s missing some libraries ?

here’s the output 

```
nononymous@Amethyst 24-03-24 macOS % ./umskt                    
dyld[10978]: Library not loaded: @rpath/lib_umskt.dylib
  Referenced from: <96491D67-CD93-30CD-9C71-876C20FB9DF2> /Volumes/OS/ISO/Soft/UMSKT/24-03-24 macOS/umskt
  Reason: tried: '/Users/runner/work/UMSKT/UMSKT/build/lib_umskt.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/work/UMSKT/UMSKT/build/lib_umskt.dylib' (no such file), '/Users/runner/work/UMSKT/UMSKT/build/_deps/fmt-build/lib_umskt.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/work/UMSKT/UMSKT/build/_deps/fmt-build/lib_umskt.dylib' (no such file), '/Users/runner/work/UMSKT/UMSKT/build/lib_umskt.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/work/UMSKT/UMSKT/build/lib_umskt.dylib' (no such file), '/Users/runner/work/UMSKT/UMSKT/build/_deps/fmt-build/lib_umskt.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/runner/work/UMSKT/UMSKT/build/_deps/fmt-build/lib_umskt.dylib' (no such file), '/usr/local/lib/lib_umskt.dylib' (no such file), '/usr/lib/lib_umskt.dylib' (no such file, not in dyld cache)
zsh: abort      ./umskt
```

also i have OpenSSL 3.2.1

**Comment by Wannabe99** [comment]
Latest 32-bit Windows build fails after Merge pull request #92… related to this? Something about OpenSSL..

**Comment by thepwrtank18** [comment]
OpenSSL started breaking for 32-bit after I updated it to 3.1.5, since the hoster of the OpenSSL download deletes anything that isn't the latest version (for good reason). Might be something that changed with the crypto libraries that throws it off. 64-bit still works, which is better than nothing.

**Comment by Wannabe99** [comment]
Since XP commonly is 32-bit it would be nice with 32-bit builds 😏, the error seem to be about library not being found at all?

CMake Error at C:/Program Files/CMake/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenSSL, try to set the path to OpenSSL root folder in the
  system variable OPENSSL_ROOT_DIR (missing: OPENSSL_CRYPTO_LIBRARY) (found
  version "3.1.5")
Call Stack (most recent call first):
  C:/Program Files/CMake/share/cmake-3.29/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/CMake/share/cmake-3.29/Modules/FindOpenSSL.cmake:682 (find_package_handle_standard_args)
  CMakeLists.txt:102 (FIND_PACKAGE)
